akka-testkit/src/main/scala/akka/testkit/TestActors.scala

```diff
@@ -19,6 +19,18 @@ object TestActors {
     }
   }
 
+  /**
+   * ForwardActor forwards all messages as-is to specified ActorRef.
+   *
+   * @param ref target ActorRef to forward messages to
+   */
+  class ForwardActor(ref: ActorRef) extends Actor {
+    override def receive = {
+      case message ⇒ ref forward message
+    }
+  }
+
   val echoActorProps = Props[EchoActor]()
+  val forwardActorProps: (ActorRef ⇒ Props) = Props(classOf[ForwardActor], _)
```

Comment:
> but a new Function1 every time an equivalent def is called.

What do you mean? There will be no Function1 allocated at all. (There will be one allocated when the def is used in a closure, but only at that site, and that is not a common thing to do with props factories).